image/cinc/install_fedora_pkg.sh

```diff
@@ -53,6 +53,7 @@ dnf -y --skip-broken install \
   python3-devel \
   python3-pip \
   python3-psutil \
+  python3-six \
```

Comment:
This changes the behavior of the fedora build.

I understand that you want to move the `pip3 -qq install six` away from the common part above, but is it desired/necessary to change the behavior for the existing OS support? If so, should it perhaps be a parent commit with an explanation?

Reply:
I saw this as an inconsistency and I just thought I'd fix it. Other python packages are installed in install script using `dnf`/`apt`, it's just this one that's installed explicitly in the Dockerfile. The package versions match in all three sources `pypi`/`apt`/`dnf` for current releases and I did not observe any behavior change. But you are right, perhaps I should move it to a separate commit.